runtime/doc/todo.txt

```diff
@@ -3158,8 +3140,7 @@ Performance:
 8   When displaying a space with only foreground highlighting, it's the same
     as a space without attributes.  Avoid displaying spaces for the "~" lines
     when starting up in a color terminal.
-8   Avoid alloc() for scratch buffer use, esp. in syntax.c.  It's very slow on
-    Win16.
+8   Avoid alloc() for scratch buffer use, esp. in syntax.c.
```

Comment:
Nah, I suppose not. Since we don't know that this is an actual issue, I'll remove it.